addons/calendar/data/mail_template_data.xml

```diff
@@ -24,7 +24,7 @@
 
         <t t-if="is_online and target_customer">
             Your appointment <strong t-out="object.event_id.appointment_type_id.name or ''">Schedule a Demo</strong> <t t-if="object.event_id.appointment_type_id.category != 'custom' and object.event_id.appointment_type_id.schedule_based_on == 'users'"> with <t t-out="object.event_id.user_id.name or ''">Ready Mat</t></t> has been booked.
-            <t t-if="is_online and object.state != 'accepted' and object.event_id.appointment_type_id.resource_manual_confirmation">
+            <t t-if="is_online and object.state != 'accepted' and object.event_id.appointment_type_id == 'resources' and object.event_id.appointment_type_id.resource_manual_confirmation">
```

Comment:
It should be `object.event_id.appointment_type_id.schedule_based_on == 'resources'`.
Also same comment for the commit as the enterprise PR, the header should include the module impacted by the changes. You can also add a small commit message (here it's void ^^) explaining the initial issue with the `opw-3603634` at the end